Dockerfile

```diff
@@ -4,10 +4,17 @@ COPY . .
 RUN make build
 
 FROM quay.io/centos/centos:stream9
-LABEL maintainers="The KubeVirt Project <kubevirt-dev@googlegroups.com>"
-LABEL description="KubeVirt CSI Driver"
+ARG git_url=https://github.com/kubevirt/cdi-driver.git
```

Comment:
There's a typo here. i think "cdi-driver" should be "csi-driver"

Reply:
I'm going to go ahead and merge this PR, then follow-up with an update to fix this typo so we can get this in asap.